src/sections/navbar.html

```diff
@@ -0,0 +1,31 @@
+<div class="flex justify-between bg-PrimaryColor p-8 sticky w-full top-0 z-10">
+  <div class="flex flex-col text-smallerSize md:text-2xl">
+    <h1>DE</h1>
+    <h1>EN</h1>
+  </div>
+  <div class="text-3xl font-bold relative lg:text-5xl">
+    <img
+      src="/src/sections/img/Vector-5.webp"
+      alt="menu-image"
+      class="md:hidden w-48 h-14"
+    />
+    <p
+      class="absolute left-4 top-2 text-smallSize md:text-5xl md:relative lg:text-5xl"
+    >
+      Baumeister
+    </p>
```

Comment:
the font-bold class in this div is affecting the font weight of the Baumeister paragraph when in reality that paragraph's font-weight is semi bold